app/src/main/java/movierating/models/Movie.java

```diff
@@ -0,0 +1,55 @@
+package movierating.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Movie {
+
+    private final String movieName;
+    private final int releaseYear;
+    private final List<Review> reviewList;
+    private final String genre;
```

Comment:
Can a movie have multiple genres? Just a doubt
Sure we can include multiple genres at later stage. We can build with single genre to simply assumptions and fast prototyping.

Reply:
Leaving this as it is for now